tests/testthat/testCombine.R

```diff
@@ -0,0 +1,60 @@
+# testCombine.R
+
+context("test for Combine SNV and CNA files")
+
+# ==== BEGIN SETUP AND PREPARE =================================================
+# set up tempdir and tempfiles for output files and to be deleted after tests
+OLOG <- as.character(getOption("rete.logfile"))   # save original logfile name
+logFileName(fPath = tempdir(), setOption = TRUE)  # make tempdir() the log dir
+# ==== END SETUP AND PREPARE ===================================================
+
+test_that("parameter errors are correctly handled", {
+    # try no parameter input
+    expect_error(combineSNV_CNA(writeLog=FALSE))
+    # try length-zero or NuLL input vector
+    input_vector <- character()
+    expect_error(combineSNV_CNA(input_vector, writeLog=FALSE))
+    # try wrong file format (e.g. text file instead of rds dataframe) in input vector
+    input_vector[1] <- "CNA.txt"
+    expect_error(combineSNV_CNA(input_vector, writeLog=FALSE))
+    # try invalid output fgX file name (e.g. wrong extension)
+    input_vector[1] <- "CNA.rds"
+    expect_error(combineSNV_CNA(input_vector, "out.txt", writeLog=FALSE))
+})
+
+
+test_that("a sane input gives an expected output", {
```

Comment:
Where are these files written to? You **have** to use ```testdir()``` and or ```testfile()``` **and** clean up after your tests. (again: cf. testImport.R)